src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -1195,15 +1205,16 @@ private function step_in_head(): bool {
 
 			/*
 			 * > A start tag whose tag name is "template"
+			 *
+			 * @todo Could the adjusted insertion location be anything other than the current location?
```

Comment:
Good question. I've read the spec several times and I don't see how. I'm concerned that means we're missing something subtle 😕 

I did check html5ever tree construction, [and it does not seem to do any special adjustment which makes me more confident.](https://github.com/servo/html5ever/blob/fd246b368e4dc9e6ae241051de45dd535af8d76e/html5ever/src/tree_builder/rules.rs#L148-L155)